Refresh for 2017 July review

timj opened the pull request.

kelseymh committed:
Expand secondary indexing section 8.7 with a performance-test figure.

kelseymh committed:
Add paragraph describing strategy for split loading of secondary index.

andy-slac committed:
Add section on asynch queries; update metadata description

jgates108 committed:
Updated shared scan entries.

jgates108 committed:
Made requested edits.

jgates108 committed:
Changed chunk ID to chun id.

abh3 committed:
Update XRootD sections and add memman/cmsd information.

abh3 committed:
Make changes suggested by John Gates.

ktlim committed:
Convert to LaTeX.

fritzm committed:
Restore Travis creds from master

womullan reviewed (commented):
async should not require a special syntax .. a service perhaps but the SQL should be the same aprt form "requiring" an into clause to say where results go.

timj committed:
Update texmf to get toc fix

ktlim committed:
Wrap long verbatim output.

ktlim committed:
Take care of \hrefs.

ktlim committed:
Extract references from ReST version.

ktlim commented:
@womullan That "INTO", plus things like figuring out how to get back a handle to inquire about query status, are exactly the syntax changes that need to be added, I believe.

timj committed:
Merge pull request #11 from lsst/u/ktl/convert-to-tex

Convert to LaTeX.

Still needs some work but the initial port is complete.

timj committed:
Move section 5, and appendices A, C and E to DMTN-046

timj committed:
Move Appendx C (InfiniDB) to DMTN-047

timj committed:
Move Appendix D to confluence

See https://confluence.lsstcorp.org/display/DM/Qserv-related+Research+Topics

timj committed:
Some references cleanup

timj committed:
Sort out final citation issues

Drizzle seems to be dead.

timj committed:
Remove urls that are not used from bib file

timj committed:
Fix ref to DMTN-046

timj committed:
Use Section not Chapter

timj committed:
Update texmf for new refs

jonathansick committed:
Add credentials for lander deployment

jonathansick commented:
I've set up the PDF document landing page: https://ldm-135.lsst.io/v/draft/index.html

timj committed:
Move Large-Scale tests to DMTR-21

The 300 node was already in DMTR-12

timj committed:
Update for DMTR-21

timj committed:
Move Design Trade Offs to DMTN-048

timj committed:
Move Other Demonstrations to DMTN-048

timj committed:
Clickable docs in change record

timj committed:
Minor cite cleanups

timj committed:
Couple more refs

fritzm committed:
Update "Current Status" and "Open Issues" sections

fritzm committed:
Spelling correction pass

fritzm committed:
Reorder sections: Intro, Reqs, Arch, Impl, Risks

fritzm committed:
Cleanup/update Implementation section

fritzm committed:
Updates to Current Status, Risks sections

- DRY (don't repeat yourself) reduce Current Status section after
previous update pass through Implementation section

- Remove outdated info from Risks section

fritzm committed:
Minor cleanup of Architecture section

fritzm committed:
Lose iMac in shared-nothing figure

fritzm committed:
Update sequencing in Introduction to match re-ordered sections

fritzm committed:
Condense Requirements section and further fixup on Summary

ktlim committed:
Make L1DB description consistent with DPDD.

ktlim committed:
Fix reference.

ktlim committed:
Fix IO typos.

ktlim committed:
Fix blatant grammar errors.

ktlim committed:
Make table names consistent with baseline schema.

ktlim committed:
Fix more grammar stumbles.

ktlim committed:
Fix lines split at trailing hyphens.

ktlim committed:
Fix more typos and grammar.

fritzm committed:
Rework Executive Summary to match revised document

fritzm committed:
Fix unescaped &

ktlim committed:
Remove unneeded forward reference.

ktlim committed:
Fix wording problems.

fritzm committed:
Address feedback/comments from Wil

fritzm committed:
Merge branch 'u/fritzm/draft-dm-review' into draft

fritzm committed:
Fix incorrect expansion of "MPP"

fritzm committed:
Change shared-nothing figure from PNG to PDF

fritzm committed:
Add drawing source for shared-nothing diagram

timj committed:
Fix Robyn's name again

timj committed:
Prepare for release
